trymito.io/components/LogoSection/LogoSection.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import logoSection from './LogoSection.module.css'
+import { log } from 'console';
+
+const RotatingLogo = (props: {imgs: {src: string, alt: string}[]}): JSX.Element => {
+    const [currentImg, setCurrentImg] = React.useState(0);
+
+    useEffect(() => {
+        const randomDelay = Math.max(Math.random() * 5000 + 2500, 2500);
+        const timeout = setTimeout(() => {
+            console.log('setting current img', currentImg);
+            setCurrentImg((currentImg + 1) % props.imgs.length);
+        }, randomDelay);
+        return () => clearTimeout(timeout);
+    }, [currentImg]);
+
+    // Force image to a fixed size, so that the container doesn't resize when the image changes
+    return (
+        <div className={logoSection.rotating_logo_container}>
+            <img className={logoSection.logo} src={props.imgs[currentImg].src} alt={props.imgs[currentImg].alt} width={150}/>
+        </div>
+    )
+}
+
+
+const LogoSection = (props: {}): JSX.Element => {
```

Comment:
This is dope. Having these up at the top is great. 

I think we could further improve this by:
1. Adding a little animation like Stremalit has on their website. The flickering is a little bit unprofessional looking. 
2. Moving the random delay counter to the Logo section and then randomly choosing which of the rotating logos (1, 2, 3, or all, etc.) to update. That way they aren't like out of sync updating very closely to each other but not exactly at the same time.